backend/case/case-persister.yml

```
@@ -520,4 +766,46 @@ serviceMaps:
+      auditAttributeBasePath: cases
+      queryMaps:
+        - query: UPDATE dristi_cases SET casetitle = ?, additionaldetails = ?, lastmodifiedby = ?, lastmodifiedtime = ? WHERE id = ?;
+          basePath: cases
+          jsonMaps:
+            - jsonPath: $.cases.caseTitle
+            - jsonPath: $.cases.additionalDetails
+              type: JSON
+              dbType: JSONB
+            - jsonPath: $.cases.auditDetails.lastModifiedBy
+            - jsonPath: $.cases.auditDetails.lastModifiedTime
+            - jsonPath: $.cases.id
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)